### ISSUE TITLE
Combining all the three datasets

### ISSUE THREAD

**Issue by Ashwin-Ramesh2607** [mks0601/PoseFix_RELEASE#42]
Hi @mks0601 , I would like to implement training on this repo, but instead of a standalone dataset. I wish to combine all the 3 datasets - COCO, MPII and PoseTrack. I want to know whether that is possible? My doubt is that COCO and PoseTrack have different keypoints. Any ideas?

**Comment by mks0601** [comment]
You can define some reference joint set (maybe a union set of various joints sets?) and use this function (https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/15446c408f664702b348eb5a7f75250b28d14ed6/common/utils/pose_utils.py#L86)

**Comment by Ashwin-Ramesh2607** [comment]
@mks0601 Thanks for that function! So coco has 17 unique keypoints, and PoseTrack has 15 unique keypoints. 13 keypoints are common between the two. The union set will therefore contain 19 keypoints. So you're suggestion is to train a model with 19 keypoints, am I correct?

**Comment by mks0601** [comment]
Yes. And you can make the loss of joints that are not intersection of the joint sets zero.
